extensions/cornerstone/src/init.tsx

```diff
@@ -296,64 +304,77 @@ export default async function init({
+      // Add to interaction requestMap
+      imageLoading.scrollInteraction(
+        scrollEvt.detail,
+        interactionPriorityCounter--,
+        imageIds,
+        viewportId
+      );
+    });
+
+    imageLoading.prefetchEnable({ uid: viewportId, imageIds }, priorityCounter--);
+    // stackPrefetch.enable({ uid: viewportId, imageIds });
+    // const { element } = evt.detail;
+    // cornerstoneTools.utilities.stackContextPrefetch.enable(element);
   });
   eventTarget.addEventListener(EVENTS.IMAGE_LOAD_FAILED, imageLoadFailedHandler);
   eventTarget.addEventListener(EVENTS.IMAGE_LOAD_ERROR, imageLoadFailedHandler);
 
-  function elementEnabledHandler(evt) {
-    // enabledElementCounter++;
-    // if (enabledElementCounter === cornerstone.getEnabledElements().length) {
-    //   const dataSource = extensionManager.getActiveDataSource()[0];
-    //   const { viewports } = viewportGridService.getState();
-    //   const viewportDisplaySetIds = [];
-    //   viewports.forEach(viewport => {
-    //     viewport.displaySetInstanceUIDs.forEach(displaySetInstanceUID => {
-    //       viewportDisplaySetIds.push(displaySetInstanceUID);
-    //     });
-    //   });
-    //   const viewportInfos = Array.from(viewports.values());
-    //   const displaySetInstanceUID = viewportInfos[0].displaySetInstanceUIDs[0];
-    //   if (displaySetInstanceUID) {
-    //     const displaySet = displaySetService.getDisplaySetByUID(displaySetInstanceUID);
-    //     if (displaySet) {
-    //       const { StudyInstanceUID } = displaySet;
-    //       displaySetService
-    //         .getActiveDisplaySets()
-    //         .filter(
-    //           ds =>
-    //             ds.StudyInstanceUID === StudyInstanceUID &&
-    //             !viewportDisplaySetIds.includes(ds.displaySetInstanceUID)
-    //         )
-    //         .sort((a, b) => a.SeriesNumber - b.SeriesNumber)
-    //         .filter(ds => !STUDY_STACKS.some(stack => stack.uid === ds.displaySetInstanceUID))
-    //         .forEach((displaySet, index) => {
-    //           const imageIds = dataSource.getImageIdsForDisplaySet(displaySet);
-    //           STUDY_STACKS.push({
-    //             uid: displaySet.displaySetInstanceUID,
-    //             imageIds,
-    //           });
-    //         });
-    //     }
-    //   }
-    //   setTimeout(() => {
-    //     console.log('All elements are enabled. Start prefetching images.');
-    //     STUDY_STACKS.forEach(stack => stackPrefetch.enable(stack, priorityCounter));
-    //   }, 2000);
-    //   eventTarget.removeEventListener(EVENTS.ELEMENT_ENABLED, elementEnabledHandler);
-    // }
-    const { element } = evt.detail;
-
-    element.addEventListener(EVENTS.CAMERA_RESET, resetCrosshairs);
-
-    eventTarget.addEventListener(EVENTS.STACK_VIEWPORT_NEW_STACK, toolbarEventListener);
-
-    initViewTiming({ element, eventTarget });
-  }
+  // function elementEnabledHandler(evt) {
```

Comment:
is this the alternate loading bar code? Might be nice to just have at the top
```
// DEBUG: set this to true for debugging locally
const debugLoadingBar = false
.
.
if debugLoadingBar {
   ... this code
}
```

so others can just update the variable to easily enable it.

Reply:
This was wen's code that we commented out for the revert. I may just delete it now since we have it in commit history but wasn't sure if there was something useful there he wanted to come back to in the future.